src/Formatter/GitBlacklistFormatter.php

```diff
@@ -45,7 +45,7 @@ private function formatOutput(string $output): string
     {
         $result = static::RESET_COLOR;
         foreach (array_filter(explode("\n", $output)) as $lineNumber => $line) {
-            $result .= preg_match('/^[0-9]+/', $line) ? $this->trimOutputLine($line, (int) $lineNumber) : $line;
+            $result .= preg_match('/^\d+/', $line) ? $this->trimOutputLine($line, (int) $lineNumber) : $line;
```

Comment:
I would prefer [0-9] here since it is easier to folow then the \d alternative.

Reply:
Done